packages/api/app/Controllers/Http/UploadController.js

```diff
@@ -1,12 +1,20 @@
+/* eslint-disable no-underscore-dangle */
 const Helpers = use('Helpers');
 const Upload = use('App/Models/Upload');
-const fs = require('fs').promises;
+const fs = require('fs');
+const fsp = require('fs').promises;
```

Comment:
```suggestion
const fs = require('fs/promises');
const { createReadStream } = require('fs');
```
Now it looks more readable 🙂